packages/runtime/test/src/interpreter/interpreter.ts

```diff
@@ -198,6 +198,7 @@ describe("Interpreter", function () {
 				setUpInterpreter(7, 1e9);
 			});
 			it("Should not throw an error when accessing account that are in transaction's account field", function () {
+				// in which exact part are we accesing the account???
```

Comment:
should it be there..?

Reply:
you are right, I just could not figure out what is going on here and left it there to ask the rest of the team. @thdailong could you provide some info about it?